test/Microsoft.OpenApi.Readers.Tests/ReferenceService/TryLoadReferenceV2Tests.cs

```diff
@@ -25,7 +25,7 @@ public void LoadSchemaReference()
             var diagnostic = new OpenApiDiagnostic();
             RootNode rootNode;
 
-            using (var stream = File.OpenRead(Path.Combine(SampleFolderPath, "multipleReferences.v2.yaml")))
+            using (var stream = Resources.GetStream(SampleFolderPath + "multipleReferences.v2.yaml"))
```

Comment:
>(SampleFolderPath + "multipleReferences.v2.yaml")) [](start = 51, length = 50)

Path.Combine is usually a preferred way to combine paths so that you don't need to worry about the trailing/beginning slashes. 

Not that big of a deal though.

Reply:
Changed.